interfaces/HO-Portal/src/app/program/disable-registration/disable-registration.component.html

```diff
@@ -1,10 +1,6 @@
-<div
-  style="display: flex;"
-  class="ion-align-items-center ion-justify-content-end"
->
-
-<ion-item *ngIf="program">
+<ion-row *ngIf="program" class="ion-align-items-center">
   <ion-label>{{ 'page.program.phases.registrationValidation.toggleText-enable-registrations'|translate }} </ion-label>
-  <ion-toggle  [ngModel]=publishedStatus (click)="updateRegistrationStatus()"></ion-toggle>
-</ion-item>
-</div>
+  <ion-toggle  [ngModel]=publishedStatus (click)="updateRegistrationStatus()" color="secondary"></ion-toggle>
```

Comment:
```suggestion
  <ion-toggle
    [(ngModel)]="publishedStatus"
    (click)="updateRegistrationStatus()"
    color="secondary"
  ></ion-toggle>
```

I'm not sure if the `ngModel`-property would need the 2-way-data-binding, but I think it almost always is for these type of properties..